lib.Makefile

```diff
@@ -203,6 +203,48 @@ define build_cgo_boring_binary
             && go tool nm $(2) | grep '_Cfunc__goboringcrypto_' 1> /dev/null'
 endef
 
+# Build a static binary with boring crypto support.
```

Comment:
If we don't want FIPS, we probably don't want boringssl/cgo either. We can use the pure-go TLS implementation. I believe the function `build_binary` defined a few lines below is doing so. Why do we want the two new definitions with boring/cgo but no FIPS?